roles/audit_storage/molecule/default/converge.yml

```diff
@@ -1,9 +1,11 @@
 ---
 - name: Converge
   hosts: all
+  vars:
+    repository_acs_version: 23.4.1
   roles:
     - role: activemq
     - role: elasticsearch
       vars:
-        elasticsearch_major_version: "{{ '8.x' if acs.version is version('25.0', 'ge') else '7.x' }}"
+        elasticsearch_major_version: "{{ '8.x' if repository_acs_version is version('25.0', 'ge') else '7.x' }}"
```

Comment:
not sure how much is worth having this condition when we are running role tests with a single version